docs/src/contribute/pull-requests.md

```diff
@@ -91,8 +91,6 @@ fix: Semi rule incorrectly flagging extra semicolon
 chore: Upgrade Esprima to 1.2, switch to using comment attachment
 ```
 
-The commit message format is important because these messages are used to create a changelog for each release. The tag and issue number help to create more consistent and useful changelogs.
-
```

Comment:
At this point, the issue number is not known yet. This paragraph deals with squashed commit messages that are created when a PR is merged, so I moved it into Step 7.

Reply:
I think "issue number" here refers to the issue the PR fixes/refs, not to the PR number, so it is known at this point (if the PR addresses an issue).

Reply:
Not sure. Our [changelogs](https://github.com/eslint/eslint/blob/v8.53.0/CHANGELOG.md) only include the PR number, not the number of the referenced/fixed issue(s). I get that squashed commit messages include both numbers, but either way, the current wording doesn't fit.

As a note, some other projects do actually include the numbers of referenced issues in their changelogs, for example c8 ([link](https://github.com/bcoe/c8/blob/main/CHANGELOG.md)).

Reply:
We used to have the number of the referenced/fixed issue in the first line (e.g., `feat: support for private-in syntax (fixes #14811)`). Either way, seems fine to remove details, and since we are mentioning the changelog later seems fine to remove this altogether from this place.